infrastructure/pipeline_2025-01-20_stack/providers.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_providers {
+    ec = {
+      source = "elastic/ec"
+    }
+    elasticstack = {
+      source = "elastic/elasticstack"
+    }
+  }
+}
+
+provider "elasticstack" {
```

Comment:
So long as this provider is within the module there will be issues deleting instances of it without a targeted destroy. Unfortunately because provisions elasticsearch users seems to require its own provider we may be stuck with this problem. 

There's no need to create a copy of the pipeline stack module without any changes, in which case just create another instance of the existing module.